### PR TITLE
B2 78 cleanup

### PR DESCRIPTION
**Pull request by nelsW70**
PR:
Closes [B2-78](https://b2io.atlassian.net/browse/B2-78)

- adjusted spacing differences between Built for Results and Designed for Humans
- changed .heading class to .header as it didn’t seem to be being applied to anything in Designed for Humans
- removed text-shadow from hero text
- update Featured Case Study four panel images with new images from Bonfire
- added border-radius and box-shadow to four panel images

Adding images: 
<details>
<summary>text spacing alignment</summary>

Screen Shot 2021-05-05 at 10 29 12 AM

</details>

### CODE REVIEW

**Review by asingleton6 — APPROVED** [review]
Looks good!

**Comment by nelsW70** [comment]
> Looks good!

Thanks Ashley!

**Review by tborres — APPROVED** [review]
Looks good. Just a minor css shorthand suggestion.

File: src/components/HomePage/DesignedForHumans.tsx [hunk 1]
```diff
@@ -67,13 +67,13 @@ const Content = styled.div`
...
 
-  .heading {
+  .header {
     margin-bottom: ${theme.spacing.sm};
     max-width: 36.75rem;
   }
 
   .tagline {
-    margin: 0 0 0.125rem;
+    margin: 0 0 ${theme.spacing.xxs} 0;
```
[the remark this answers is not in the document]
Comment (nelsW70) [reply src/components/HomePage/DesignedForHumans.tsx:76]: Ah, thanks Tony!

**Pull request merged**